bosh-release/setup.sh

```diff
@@ -0,0 +1,266 @@
+#!/bin/bash
```

Comment:
This looks impressive! Great work! May be we can add license headers to these files. Thanks!